src/Stormpath/Exceptions/UserAgentException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace Stormpath\Exceptions;
+
+class UserAgentException extends \Exception {};
```

Comment:
We don't need a new exception for this, we can just use an `IllegalStateException` or `IllegalArgumentException` to avoid creating a new exception for every case where we need to throw.